### PR TITLE
🐛fix wrong tag version error

### PR DESCRIPTION
**Pull request by weiye-Lee**
## What type of PR is this?
- [x] bug
- [ ] feature
- [ ] enhancement

#### Issue(s) number: 
👀

#### Description:
🐛fix the wrong tag version error in docker-compose file

## How do you solve it?
👀

## Special notes for your reviewer, ex. impact of this fix, design document, etc:
👀

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -325,7 +325,7 @@ services:
       - SYS_PTRACE
   
   console:
-    image: docker.io/vesoft/nebula-console:3.5.0
+    image: docker.io/vesoft/nebula-console:v3.5.0
```
Comment (wey-gu) [inline docker-compose.yaml:328]: could you please put `vesoft/nebula-console:v3.5` instead? Thanks!

**Comment by wey-gu** [comment]
Thanks for the contribution! could you please help sign the CLA and patch as commented?

BR..Wey

File: docker-compose.yaml (hunk 1 above)
Comment (weiye-Lee) [reply docker-compose.yaml]: no problem😜

**Review by wey-gu — APPROVED** [review]

**Comment by wey-gu** [comment]
Thanks @weiye-Lee !

@QingZ11 we have a new contributor now!

**Pull request merged**